build.go

```diff
@@ -935,9 +936,12 @@ func NewGometalinter(linter string, dirs ...string) *gometalinter {
 }
 
 func (g *gometalinter) verifyInstalled() {
-	if _, err := runError(g.command, "--disable-all"); err != nil {
-		log.Fatalf("gometalinter is not installed")
+	if !g.verifiedInstallation {
+		if _, err := runError(g.command, "--disable-all"); err != nil {
+			log.Fatalf("gometalinter is not installed")
+		}
 	}
+	g.verifiedInstallation = true
```

Comment:
no, due to creating new instances, this is b*shit. I need to fix it. Too late yesterday.

I'd like to keep it encasulated and maybe checked multiple times. This should be a performance penalty of 200ms(?)  each run. Else it become cluttered. 

If it wasn't in the build tool it would have been a different thing.